scripts/rollup/build.js

```diff
@@ -155,8 +156,10 @@ function getBabelConfig(
     packageName === 'react' || externals.indexOf('react') !== -1;
   let options = {
     exclude: '/**/node_modules/**',
+    babelHelpers: 'bundled',
```

Comment:
- `@rollup/plugin-babel`: https://github.com/rollup/plugins/tree/master/packages/babel
  - `babelHelpers`: https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers
    - > Type: `'bundled' | 'runtime' | 'inline' | 'external'`
    - > Default: `'bundled'`
    - > It is recommended to configure this option explicitly (even if with its default value) so an informed decision is taken on how those babel helpers are inserted into the code.
    - > `'bundled'` - you should use this if you want your resulting bundle to contain those helpers (at most one copy of each). Useful especially if you bundle an application code.